pkg/apis/infinispan/v1/types_util.go

```diff
@@ -369,3 +371,69 @@ func (ispn *Infinispan) StorageSize() string {
 	}
 	return ""
 }
+
+// AddLabelsForPods adds to the user maps the labels defined for pods in the infinispan CR
+func (ispn *Infinispan) AddLabelsForPods(uMap map[string]string) {
+	addLabelsFor(ispn, "infinispan.org/podTargetLabels", uMap)
```

Comment:
We should keep markers in `constants`, there a big chance to make typo

Reply:
done